mne_bids/tests/test_dig.py

```diff
@@ -334,7 +334,7 @@ def test_electrodes_io(tmp_path):
     raw.pick_types(eeg=True, stim=True)  # we don't need meg channels
     bids_root = tmp_path / 'bids1'
     bids_path = _bids_path.copy().update(root=bids_root, datatype='eeg')
-    os.makedirs(op.join(bids_root, 'sub-01', 'ses-01', 'eeg'), exist_ok=True)
+    (bids_root / 'sub-01' / 'ses-01' / 'eeg').mkdir(exist_ok=True)
```

Comment:
Why do you need to create this directory manually, anyway..?

Reply:
I'm not sure.. I was having a hard time understanding this, but was trying to follow the examples from the other tests in `test.dig.py`

Reply:
The `write_raw_bids` should create the directory tree, could you try without?